src/languages/bash.js

```diff
@@ -69,6 +69,11 @@ export default function(hljs) {
     begin: /'/,
     end: /'/
   };
+  const ESCAPED_APOS = {
+    className: '',
+    begin: /\\'/
+
```

Comment:
Remove extra blank line please.

Reply:
Thank you for the review. I just want to say, that the code is a remake of `const ESCAPED_QUOTE` above. The `ESCAPED_QUOTE` also has the  `className: ''`, the `begin` and even the blank line. If you want I can refactor it, in this pull request.

Reply:
> If you want I can refactor it, in this pull request.

Sure.